talon.wdl

```diff
@@ -446,39 +446,38 @@ task SummarizeDatasets {
 
 task Talon {
     input {
-        File SAMfile
         File configFile
         File databaseFile
         String genomeBuild
         Float minimumCoverage = 0.9
         Int minimumIdentity = 0
         String outputPrefix
         String configFileName = basename(configFile)
-        String SAMfileName = basename(SAMfile)
 
         Int cores = 4
-        String memory = "20G"
+        String memory = "25G"
         String dockerImage = "biocontainers/talon:v4.4.1_cv1"
     }
 
     command {
         set -e
         mkdir -p $(dirname ~{outputPrefix})
         mv ${configFile} ./${configFileName}
```

Comment:
Why is this `mv` needed?

Reply:
Ah yes thank you, that was a left-over that should now be fixed by export TMPDIR=/tmp.